src/query/anonymization.c

```diff
@@ -433,18 +433,7 @@ static seed_t prepare_bucket_seeds(Query *query, ParamListInfo bound_params)
 
 static bool is_datetime_type(Oid type)
 {
-  switch (type)
-  {
-  case DATEOID:
-  case TIMEOID:
-  case TIMETZOID:
-  case TIMESTAMPOID:
-  case TIMESTAMPTZOID:
-    return true;
-  default:
-    /* Slower to evaluate, safety net in case new types get added. */
-    return TypeCategory(type) == TYPCATEGORY_DATETIME;
-  }
+  return TypeCategory(type) == TYPCATEGORY_DATETIME;
```

Comment:
Makes more sense now to inline this call. The comparison is pretty explicit by itself.